domain/domain.go

```diff
@@ -184,6 +184,7 @@ func (do *Domain) loadInfoSchema(startTS uint64) (infoschema.InfoSchema, bool, i
 		loadSchemaDurationTotal.Observe(time.Since(beginTime).Seconds())
 	}()
 	snapshot := do.store.GetSnapshot(kv.NewVersion(startTS))
+	snapshot.SetOption(kv.TiKVClientReadTimeout, uint64(3000)) // 3000ms.
```

Comment:
What is the default value if not set?

Reply:
@zimulala 
The default value of `Get` is 30s and `BatchGet` is 60s.

Reply:
If `snapshot.SetOption(kv.ReplicaRead, kv.ReplicaReadMixed)` is not set, the leader peer would be tried first. If timeout happens the [error is cached](https://github.com/tikv/client-go/blob/tidb-6.5-with-kv-timeout-feature/internal/locate/region_request.go#L1569), the replica selector [would be transferfred to](https://github.com/tikv/client-go/blob/tidb-6.5-with-kv-timeout-feature/internal/locate/region_request.go#L913) `TryFollower` automatically , then follower read [would be used](https://github.com/tikv/client-go/blob/tidb-6.5-with-kv-timeout-feature/internal/locate/region_request.go#L439) on the other peers.

So it's not necessary to involve follower read in the first try I think.

Reply:
@Tema 

The fast automatic fallback happens only when the timeout is configured to a value smallter than the default value, [here](https://github.com/tikv/client-go/blob/tidb-6.5-with-kv-timeout-feature/internal/locate/region_request.go#L901) is the check logic.
In the default path, the retry may happen at most `maxReplicaAttempt` times.